class-03-responsive-mobile-first/pair-assignment/zach-jeff/starter-code/styles/modules.css

```diff
@@ -0,0 +1,76 @@
+.site-header {
+  background: #FFEAAA;
+}
+
+/* DONE: Float the main header to the left, to make room for the hamburger menu. */
+.heading-group-h1 {
+  float: left;
+  padding: 0 0 .2em 1em;
+}
+
+/* TODO: Add styles for the icon-menu class. It should be bigger, and off to the right. */
+.icon-menu {
+  position: absolute;
+  top: 3%;
+  right: 2%;
+  font-size: 300%;
+}
+
+.main-nav {
+  padding: 0.1em;
+}
+
+.main-nav ul {
+  display: none;
+  padding: 0;
+  list-style: none;
+  clear: both;
+}
+
+.main-nav:hover ul {
+  display: block;
+}
+
+.main-nav li {
+  margin-right: .4em;
+  font-size: 1.5em;
+  padding: .5em .2em;
+  border-bottom: 1px solid #ccc;
+}
+
+#filters {
+  padding: 0;
+  list-style: none;
+}
+
+.template {
+  display: none;
+}
+
+.byline {
+  color: #777;
+}
+
+.byline a {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+/* TODO: Now add styles for how things should look on a larger screen:
+         - Hide the hamburger.
+         - Main should be all on one row, with padding spaced better for a larger screen.
+         - Just, ya know, try to make it look good! */
+/* @media only screen... */
+@media only screen and (min-width: 640px) {
+  .icon-menu {
+    display: none;
+  }
+  .main-nav ul {
+    display: block;
+  }
+  .main-nav li {
+    display: inline;
+    padding: .5em 2%;
```

Comment:
You're mixing ems and percents here, which isn't strictly wrong, but it's kind of weird and will probably get you some raised eyebrows.